link-parent/link-ilp-over-http/src/main/java/org/interledger/link/http/IncomingLinkSettings.java

```diff
@@ -75,38 +75,47 @@
                   .map(Object::toString)
                   .map(String::toUpperCase)
                   .map(IlpOverHttpLinkSettings.AuthType::valueOf)
-                  .ifPresent(builder::authType);
+                  .ifPresent((authType) -> {
+                    builder.authType(authType);
+
+                    if (authType != IlpOverHttpLinkSettings.AuthType.SIMPLE) {
```

Comment:
It's not in the ticket, but we should probably do this change for the outgoing links too.